src/CSET/operators/regrid.py

```diff
@@ -55,38 +60,60 @@ def regrid_onto_cube(
     -----
     Currently rectlinear grids (uniform) are supported.
     """
-    # Get y,x coord names
-    y_coord, x_coord = get_cube_yxcoordname(incube)
-
-    # List of supported grids - check if it is compatible
-    supported_grids = (iris.coord_systems.GeogCS,)
-    if not isinstance(incube.coord(x_coord).coord_system, supported_grids):
-        raise NotImplementedError(
-            f"Does not currently support {incube.coord(x_coord).coord_system} coordinate system"
-        )
-    if not isinstance(incube.coord(y_coord).coord_system, supported_grids):
-        raise NotImplementedError(
-            f"Does not currently support {incube.coord(y_coord).coord_system} coordinate system"
-        )
-
-    regrid_method = getattr(iris.analysis, method, None)
-    if callable(regrid_method):
-        return incube.regrid(target, regrid_method())
+    # If only one cube, put into cubelist as an iterable.
+    if type(toregrid) == iris.cube.Cube:
+        toregrid = iris.cube.CubeList([toregrid])
+
+    # To store regridded cubes
+    regridded_cubes = iris.cube.CubeList()
+
+    # Iterate over all cubes and regrid
+    for cube in toregrid:
```

Comment:
```suggestion
    # To store regridded cubes.
    regridded_cubes = iris.cube.CubeList()

    # Iterate over all cubes and regrid.
    for cube in iter_maybe(toregrid):
```
You'll have to put `from CSET._common import iter_maybe` at the top.